tests/Backend/Workspaces/WorkspacesUnloadTest.php

```diff
@@ -17,6 +18,41 @@
 
 class WorkspacesUnloadTest extends WorkspacesTestCase
 {
+    public function testTableCloneCaseSensitiveThrowsUserError()
+    {
+        $tokenData = $this->_client->verifyToken();
+        if (in_array($tokenData['owner']['defaultBackend'], [self::BACKEND_REDSHIFT, self::BACKEND_SYNAPSE])) {
+            return;
```

Comment:
```suggestion
            $this->markTestSkipped("Nejaky duvod");
```

lepší je ten test označit za skipnutý